source/54-1 Timothy.usfm.db

```diff
@@ -156,8 +156,8 @@ This must be our testimony, as opportunities present themselves;
 \v 1 All who are in the position of slaves should regard their masters as deserving of the greatest respect, so that the name of God, and our teaching, may not be maligned.
 \v 2 Those who have Christian masters should not think less of them because they are [neut:also followers of Christ|masc:their brothers], but on the contrary they should serve them all the better, because those who are to benefit by their good work are dear to them as their fellow Christians.
 \s Conclusion
-Those are the things to insist on in your teaching.
 \p
+Those are the things to insist on in your teaching.
```

Comment:
Yeah, it would make sense to indicate this with another tag, but it looks like so far paragraph-spanning verses are noted without it. It looks like there are 13 other cases so far (regex search for "\\p\n\w"): Haggai 2:9; Luke 9:43; Acts 8:1, 9:40, 14:10, 16:12; 1 Cor 12:31; 2 Cor 12:9; 1 Thess 1:10, 5:5; 1 Tim 2:6; 1 Pet 5:14; 1 John 4:16.